Framework/CurveFitting/test/Algorithms/VesuvioCalculateMSTest.h

```diff
@@ -79,6 +79,8 @@ createTestWorkspace(const bool detShape = true,
       // Add another detector in the same position as the first
       auto shape = ShapeFactory().createShape(shapeXML);
       Mantid::Geometry::Detector *det2 = new Detector("det1", 2, shape, NULL);
+      // Setting detectors should normally go via DetectorInfo, but here we need
+      // to set a position as we are adding a new detector.
```

Comment:
Could you say a little more about this. Particularly about relationships with parameter maps and detector info. And say that in general one shouldn't call `setPos`.